dockstore-webservice/src/test/java/io/dockstore/webservice/languages/CWLHandlerTest.java

```diff
@@ -337,13 +354,31 @@ public void testGetContentWithMalformedDescriptors() throws IOException {
         try {
             cwlHandler.getContent("/invalidMapCWL.cwl", FileUtils.readFileToString(cwlFile, StandardCharsets.UTF_8), emptySet,
                 LanguageHandlerInterface.Type.TOOLS, toolDAO);
-            Assert.fail("Expected ($)import/($)include error");
+            Assert.fail("Expected parse error");
```

Comment:
this seems less specific, not sure this is an improvement.
What is the story behind this?

Reply:
The code is that way because I believe it to be an improvement: the original `assert` message was incorrect.  The problem in the test file is that, after preprocessing, the value of the `run` field is neither a string (file name) nor a workflow or tool.  An import or include is not required in that context.  I tweaked the message to be a little more specific.